src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -95,6 +96,20 @@ public Conversation getConversationWithTitle(String title) {
     }
     return null;
   }
+  
+  /**
+   * Access the Conversation object with the given UUID.
+   *
+   * @return null if the UUID does not match any existing Conversation.
+   */
+  public Conversation getConversationWithID(UUID id) {
+    for (Conversation conversation : conversations) {
+      if (conversation.getId().equals(id)) {
+        return conversation;
+      }
+    }
+    return null;
+  }
 
```

Comment:
I think this method would be particularly helpful with accessing conversations of users & groups of users!!